spring-boot-project/spring-boot-docs/src/main/asciidoc/spring-boot-features.adoc

```diff
@@ -6423,28 +6469,28 @@ should be specified using `value` or `components` attribute of `@RestClientTest`
 	}
 ----
 
-A list of the auto-configuration that is enabled by `@RestClientTest` can be
+A list of the auto-configuration settings that are enabled by `@RestClientTest` can be
 <<appendix-test-auto-configuration#test-auto-configuration,found in the appendix>>.
 
 
 
 [[boot-features-testing-spring-boot-applications-testing-autoconfigured-rest-docs]]
-==== Auto-configured Spring REST Docs tests
-The `@AutoConfigureRestDocs` annotation can be used if you want to use Spring REST Docs
-in your tests with Mock MVC or REST Assured. It removes the need for Spring REST Docs'
-JUnit rule.
+==== Auto-configured Spring REST Docs Tests
+You can use The `@AutoConfigureRestDocs` annotation to use Spring REST Docs
```

Comment:
"The" -> "the"

Reply:
Yes. Thanks. (The perils of rearranging to have an active voice.)